lisa/tools/fio.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import TYPE_CHECKING, List, Type, cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+
+from .git import Git
+
+if TYPE_CHECKING:
+    from .make import Make
```

Comment:
Does it work in the TYPE_CHECKING? the Make is used in the logic as a type. It may cause error on runtime.

Reply:
not test this path yet, without this, it will enter into circular import error.